src/cli/dcos_docker/commands/create.py

```diff
@@ -456,6 +483,16 @@ def create(
     )
 
     try:
+        if loopback_device is not None:
```

Comment:
This block will not raise a `CalledProcessError`, right? If so, it should be outside of the `try` block.